modules/create-additional-routes-ocp-oauth.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-ocp-oauth.adoc
+:_module-type: PROCEDURE
+[id="create-additional-routes-ocp-oauth_{context}"]
+= Creating additional routes for {ocp} OAuth server
+
+When you configure {ocp} OAuth server as an identity provider by using {product-title} portal, {product-title-short} configures only a single route for the OAuth server.
+However, you can create additional routes by specifying them as annotations in the Central custom resource.
+
+.Prerequisites
+
+* You must have configured link:https://docs.openshift.com/container-platform/4.11/authentication/using-service-accounts-as-oauth-client.html#service-accounts-as-oauth-clients_using-service-accounts-as-oauth-client[Service accounts as OAuth clients] for your {ocp} OAuth server.
+
+.Procedure
+
+* If you installed {product-title-short} using the {product-title-short} Operator:
+. Create a `CENTRAL_ADDITIONAL_ROUTES` environment variable that contains a patch for the Central custom resource:
++
+[source,terminal]
+----
+$ CENTRAL_ADDITIONAL_ROUTES='
+spec:
+  central:
+    exposure:
+      loadBalancer:
+        enabled: false
+        port: 443
+      nodePort:
+        enabled: false
+      route:
+        enabled: true
+    persistence:
+      persistentVolumeClaim:
+        claimName: stackrox-db
+  customize:
+    annotations:
+      serviceaccounts.openshift.io/oauth-redirecturi.main: sso/providers/openshift/callback <1>
+      serviceaccounts.openshift.io/oauth-redirectreference.main: "{\"kind\":\"OAuthRedirectReference\",\"apiVersion\":\"v1\",\"reference\":{\"kind\":\"Route\",\"name\":\"central\"}}" <2>
+      serviceaccounts.openshift.io/oauth-redirecturi.second: sso/providers/openshift/callback <3>
+      serviceaccounts.openshift.io/oauth-redirectreference.second: "{\"kind\":\"OAuthRedirectReference\",\"apiVersion\":\"v1\",\"reference\":{\"kind\":\"Route\",\"name\":\"second-central\"}}" <4>
+'
+----
+<1> The redirect URI for setting the main route.
+<2> The redirect URI reference for the main route.
+<3> The redirect for setting the second route.
+<4> The redirect reference for the second route.
+
+. Apply the `CENTRAL_ADDITIONAL_ROUTES` patch to the Central custom resource:
++
+[source,terminal]
+----
+$ oc patch centrals.platform.stackrox.io \
+  -n <namespace> \
+  <custom-resource> \ <1>
+  --patch "$CENTRAL_ADDITIONAL_ROUTES" \
+  --type=merge
+----
+<1> Name of the Central custom resource.
```

Comment:
You could update this call out to the following:

```
<1> Replace `<custom-resource>` with the name of the Central custom resource.
```

You could also consider adding a call out highlighting the need to replace `<namespace>` with the name of the project that contains the custom resource (if my assumption is correct).

My apologies for not mentioning this in the first review.

Reply:
Alternatively, you could update the line to "The name of the Central custom resource".